opentelemetry-exporter-cloud-monitoring/src/opentelemetry/exporter/cloud_monitoring/__init__.py

```diff
@@ -1,3 +1,5 @@
+# pylint: disable=too-many-locals
```

Comment:
Why is this needed? And can you resolve this without resorting to disabling tests?

Reply:
removed it by moving to new fn